requirements/requirements.txt

```diff
@@ -1,19 +1,20 @@
+brax==0.10.3
 colorama
-distrax @ git+https://github.com/google-deepmind/distrax  # distrax release doesn't support jax > 0.4.13
+distrax
```

Comment:
Why did we remove the comment here?

Reply:
Ah because they updated the pypi :smile: